server/utils/room.js

```diff
@@ -214,32 +215,40 @@ class RoomManager {
     this.jobRunning = false
     // if the job is finished, each node stops running
     Object.keys(this.nodes).forEach((nodeId) => this.nodes[nodeId].running = false)
-
     History.create({
       nodes: Object.keys(this.nodes).length,
+      maxGen: this.maxGen,
       room: this.room,
       result: this.lastResult,
       startTime: this.start,
+      populationSize: this.populationSize,
+      fitnessGoal: this.fitnessGoal,
+      chromosomeLength: this.chromosomeLength,
+      elitism: this.elitism,
+      reproductiveCoefficient: this.reproductiveCoefficient,
       endTime: new Date(),
+      fitnessFunc: this.fitness.function.toString(),
+      mutations: map(this.mutations, mut =>
+        `${mut.name} ${mut.chanceOfMutation}`).join(','),
+      selection: this.selection.name,
+      genePool: this.genePool.join(','),
+      admins: map(this.admins, admin => `${admin.id}`).join(','),
+      totalFitness: this.totalFitness,
+      // FIXME: room stats is going to be in micro-service later igbnore for now
+      // roomStats: this.roomStats.getStats()
```

Comment:
Did you mean to leave roomStats here? Also ,we should keep fixme comments out of master according to corey / omri